README.md

```diff
@@ -64,3 +69,15 @@ pnpm start
 # or
 bun start
 ```
+
+Note: When running an iOS development build, make sure you're running the latest version on the simulator (>iOS 18).
+
+## Necessary polyfills
```

Comment:
we could add links to:

* https://docs.solanamobile.com/react-native/polyfill-guides/web3-js
* https://docs.solanamobile.com/react-native/polyfill-guides/spl-token